runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/blog/BreadcrumbsSection/BreadcrumbsSection.tsx

```diff
@@ -1,9 +1,92 @@
-import React from 'react';
+import React, { FC } from 'react';
+
+import Image from 'next/image';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+
+import robotIcon from '#public/images/icons/toy-orange.svg';
+import If from '#src-app/components/utils/If';
+import Typography from '#src-landing/components/Typography';
+import { capitalizeFirstLetter } from '#src-landing/utils/utils';
 
 import styles from './BreadcrumbsSection.module.scss';
+import { BreadcrumbsSectionProps } from './BreadcrumbsSection.types';
+
+const BreadcrumbsSection: FC<BreadcrumbsSectionProps> = ({
+    postTitle,
+}) => {
+    const router = useRouter();
+    const { pathname, asPath } = router;
+    const queryChars = ['?', '&', '='];
+    const uncoveredUrlPart = 'post';
+
+    const relevantPath = queryChars
```

Comment:
just ignore params, you dont need them. Use url without it or split by `?`

Reply:
done